### PR TITLE
Translate Synthetic Event page

### PR DESCRIPTION
**Pull request by helloanil**

### CODE REVIEW

File: content/docs/reference-events.md [hunk 1]
```diff
@@ -244,7 +242,9 @@ string pointerType
 boolean isPrimary
 ```
 
-A note on cross-browser support:
+Çapraz-tarayıcı desteği üzerine bir not:
+
+İşaretçi olayları henüz tüm tarayıcılarda desteklenmemektedir (Bu makalenin yazıldığı tarihte destekleyen tarayıcılar: Chrome, Firefox, Edge ve Internet Explorer). React kasıtlı olarak 
 
 Pointer events are not yet supported in every browser (at the time of writing this article, supported browsers include: Chrome, Firefox, Edge, and Internet Explorer). React deliberately does not polyfill support for other browsers because a standard-conform polyfill would significantly increase the bundle size of `react-dom`.
```
Comment (alioguzhan) [inline content/docs/reference-events.md]: I think this section is forgotten. should be deleted

File: content/docs/reference-events.md [hunk 2]
```diff
@@ -227,9 +225,9 @@ onLostPointerCapture onPointerEnter onPointerLeave onPointerOver onPointerOut
 
 The `onPointerEnter` and `onPointerLeave` events propagate from the element being left to the one being entered instead of ordinary bubbling and do not have a capture phase.
```
Comment (alioguzhan) [inline content/docs/reference-events.md]: should be translated or deleted?

File: content/docs/reference-events.md [hunk 3]
```diff
@@ -195,7 +193,7 @@ onMouseMove onMouseOut onMouseOver onMouseUp
 
 The `onMouseEnter` and `onMouseLeave` events propagate from the element being left to the one being entered instead of ordinary bubbling and do not have a capture phase.
```
Comment (alioguzhan) [inline content/docs/reference-events.md]: missed section.

File: content/docs/reference-events.md [hunk 4]
```diff
@@ -60,62 +58,62 @@ function onClick(event) {
...
+> Olay özelliklerine asenkron bir şekilde erişmek isterseniz, olay üzerinde `event.persist()` çağırmalısınız. Bu şekilde sentetik olay havuzdan çıkarılır ve olay referanslarının kullanıcı kodu tarafından korunmasına olanak sağlanır.
+
+## Desteklenen Olaylar {#supported-events}
+
+React olayları normalleştirir ve olaylar bu şekilde farklı tarayıcılarda tutarlı özelliklere sahip olur.
+
+Aşağıdaki olay yöneticileri, balonlanma(bubbling) evresinde bir olay tarafından tetiklenir. Yakalama evresine bir olay yöneticisi kaydetmek için, olay ismine `Capture` ilave edin. Örneğin tıklama olayını, yakalama evresinde yönetmek için `onClick` kullanmak yerine `onClickCapture` kullanın.
+
+- [Not Panosu Olayları](#clipboard-events)
+- [Dizim Olayları](#composition-events)
```
Comment (alioguzhan) [inline content/docs/reference-events.md]: In the context of this event type, the words `bilesim`, `birlesim` or even `duzenleme` might be more accurate. 

From [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CompositionEvent) -> `The DOM CompositionEvent represents events that occur due to the user indirectly entering text.`

From [StackOverflow](https://stackoverflow.com/a/51413620/1510671) ->  `Composition Events might be used to add accents to characters despite their absence from standard US keyboards, to build up logograms of many Asian languages from their base components or categories, to select word choices from a combination of key presses on a mobile device keyboard.`

What do you think ?

Comment (ozcanzaferayan) [reply content/docs/reference-events.md]: I think `Kompozisyon` can be used for `Composition`. You can check this word on [tdk](http://www.tdk.gov.tr):
image

Comment (helloanil) [reply content/docs/reference-events.md]: If TDK says so, I can use `Kompozisyon` for this to keep it simple. Thanks for your valuable comments @alioguzhan

**Pull request merged**